Backend/src/main/java/idealab/api/operations/DropboxOperations.java

```diff
@@ -131,11 +132,9 @@ public PrintJobResponse updateDropboxPath(DropBoxFilePathRequest request) {
 
     try {
       //Copy file to new location in dropbox, delete old file from dropbox, update database
-      RelocationResult result = client.files().copyV2(oldPath, newPath);
+      RelocationResult result = client.files().moveV2(oldPath, newPath.endsWith(FILE_EXTENSION) ? newPath : newPath + FILE_EXTENSION);
```

Comment:
Can you move this check to line 129 where the newPath is created?

The `if` statement on line 137 fails when the path is modified without updating the `newPath` variable.